src/style.css

```diff
@@ -1 +1,86 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir, Helvetica, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-bar--big,
+.search-bar--small {
+  margin-top: 20px;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Avenir, sans-serif;
```

Comment:
The use of the `*` selector is generally discouraged due to its impact on performance. It would be better to set these styles for specific elements that require them. (See the CSS section of the checklist)